app/src/theme/index.ts

```diff
@@ -0,0 +1,51 @@
+import { DefaultTheme, DarkTheme, useTheme as useThemeOriginal } from "@react-navigation/native";
+import Color from "./colors";
+
+
+export declare type AppTheme = {
```

Comment:
`declare` se myslím používá jenom v declaration souborech (.d.ts). V implementačním (.ts) souboru je to zbytečné, stačí

```suggestion
export type AppTheme = {
```